meta/requirements.yml

```diff
@@ -0,0 +1,17 @@
+---
+# Note that dependencies listed here are made available to the role
+# but _are not_ automatically installed.  Role variables cannot be
+# specified here.
+#
+# It _is_ possible to list both collections and roles in this file,
+# but unfortunately ansible-galaxy attempts to naively merge the
+# dependencies listed in meta/main.yml with these.  That means that
+# both sets of dependencies must be lists. :(
+#
+# See also cisagov/skeleton-ansible-role#153.
+collections:
```

Comment:
Ansible again forces us into awkward patterns for basic dependency functionality 😑

Reply:
This complication is a big part of why I advocated for defining _all_ dependencies in the `meta/requirements.yml` file. It is also why I suspect Ansible will go that direction sometime soon.  Listing role dependencies in both `meta/main.yml` and `meta/requirements.yml` isn't viable if you have any collection dependencies.

Reply:
@mcdonnnj - Can this discussion be resolved?

Reply:
See also cisagov/skeleton-ansible-role#153.